src/Tasks/AssignCulture.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using Microsoft.Build.Collections;
```

Comment:
I don't see any collection in the file changes. What is this used for please?

Reply:
The `MSBuildNameIgnoreCaseComparer` added is defined there